server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -131,11 +137,20 @@ public CobolVisitor(
               Node rootNode = new RootNode(locality, copybooks);
               visitChildren(ctx).forEach(rootNode::addChild);
               addCopyNodes(rootNode, copybooks.getUsages());
+              addFlavorNode(rootNode);
```

Comment:
Does it mean that the flavor nodes will be added only to the root node?

Reply:
I mean, by this time there is going to be only the rootNode in the AST

Reply:
The whole subtree is added under the root two lines before:
```
visitChildren(ctx).forEach(rootNode::addChild);
```

Reply:
Sorry, my bad, I didn't notice.